frontend/portals/Loader/styles.js

```diff
@@ -0,0 +1,11 @@
+import { css } from 'glamor';
+
+export default css({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  zIndex: 500,
```

Comment:
Currently the overlay overlaps everything since the zIndex is so high. With an index of `4` the popup would stay behind the payment bar which looks a little bit nicer from my perspective. The bar already takes care that the button is disabled when the cart is about to change.

Reply:
z-index is changed to 4